samples/net/lwm2m_client/src/lwm2m-client.c

```diff
@@ -177,6 +178,32 @@ static int firmware_update_cb(u16_t obj_inst_id)
 }
 #endif
 
+
+static void *temperature_get_buf(u16_t obj_inst_id, size_t *data_len)
+{
+	/* Last read temperature value, will use 25.5C if no sensor available */
+	static struct float32_value v = { 25, 500000 };
+	struct device *dev = NULL;
+
+#if defined(CONFIG_FXOS8700_TEMP)
+	dev = device_get_binding(DT_NXP_FXOS8700_0_LABEL);
+#endif
+
+	if (dev != NULL) {
+		if (sensor_sample_fetch(dev)) {
+			LOG_ERR("temperature data update failed");
+		}
+
+		sensor_channel_get(dev, SENSOR_CHAN_DIE_TEMP,
+				(struct sensor_value *) &v);
```

Comment:
Let's get this rebased on today's master and fix 1 minor nit: this line needs to be aligned with the parenthesis above it like so (added 3 spaces):
```
		sensor_channel_get(dev, SENSOR_CHAN_DIE_TEMP,
				   (struct sensor_value *) &v);
```

Sorry this took so long :/  work has been.. odd.

Reply:
Done.